client/repoutils.c

```diff
@@ -541,7 +541,9 @@ TDNFFindRepoById(
     )
 {
     uint32_t dwError = 0;
+    PTDNF_REPO_DATA parse_pRepos = NULL;
```

Comment:
Nit: please adhere to naming standards - name it `pRepoParse` or something. Use a suffix, not a prefix.

Reply:
Done